python/phylanx/ast/transformation.py

```diff
@@ -437,19 +439,24 @@ def __init__(self, f):
             # Create the AST
             tree = ast.parse(src)
             ast.increment_lineno(tree, actual_lineno)
-            transformation = self.targets[target]()
+            transformation = self.targets[target]
 
             assert len(tree.body) == 1
 
             if target == "PhySL":
                 self.__physl_src__ = '%s(%s)\n' % (
```

Comment:
Should we make sure that no key word arguments are provided for the "PhySL" backend?